### DISCUSSION TITLE
Global variables are not updated via Flows

### DISCUSSION THREAD

**Discussion by tharindraj** [postmanlabs/postman-flows#226] [Defect]
I have set of Global Variables defined and i have written in a particular request to update a Global variable once it received the response. 
And then use that for next request. When that API request is individually executed, i can see the Global variable is updated and the current value is shown correctly.
But when i execute that request via "Flows", it does not overwrite the initiate value, hence the send API call still uses the initial value

**Comment by tharindraj** [comment]
Please get this fixed soon :-(

**Reply by saswatds** [reply]
Hey @tharindraj  We do not supporting updating globals and environment using flows. We wrote a post regarding this here https://github.com/postmanlabs/postman-flows/discussions/142

Can you tell us what use-case you are trying to solve? We will try to come up with a good solution.

**Reply by prabodhprakash** [reply]
hi, the idea is simple. Flows are created from requests and it is redundant effort to manipulate variables/data just for flows, when it has already been done for requests. Sometimes the scripts to update variables are also complex...so, overall (atleast for me) - flow becomes almost unusable - if I cannot update environment variables.

**Comment by tharindraj** [comment]
Hello Saswat,

Thanks for your reply. My context is that i have an authentication API
Call. the final output i write to a Global variable and it is then used as
an Authentication token for the following calls. that does not work flows
at the moment.
2nd one is I need a variable which holds a value and increment each time i
run the request from flow. So if i run the flow again next day, it should
start incrementing from where i stopped.

I am new to Postman and Flows and need some support on getting this
clarified. can you help me with this? or any chance to arrange a session
with you?
